test/java/seedu/addressbook/data/AddressBookTest.java

```diff
@@ -11,12 +11,7 @@
 import org.junit.Test;
 import org.junit.rules.ExpectedException;
 
-import seedu.addressbook.data.person.Address;
-import seedu.addressbook.data.person.Email;
-import seedu.addressbook.data.person.Name;
-import seedu.addressbook.data.person.Person;
-import seedu.addressbook.data.person.Phone;
-import seedu.addressbook.data.person.UniquePersonList;
+import seedu.addressbook.data.person.*;
```

Comment:
Coding standard violation: [Imported classes should always be listed explicitly.](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html#package-and-import-statements)